nativelib/src/main/resources/__posix__/platform.c

```diff
@@ -0,0 +1,68 @@
+#ifdef _WIN32
```

Comment:
Why is `platform.c` in posix-only directory? It's supposed to be available on every platform.

Reply:
I think this is a Github bug, see my branch here:
https://github.com/muxanick/scala-native/tree/topic/windows_basics/nativelib/src/main/resources

Reply:
Actually, it is in both location, I would remove from posix.